packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -77,7 +77,15 @@ export class BabylonViewer extends LitElement {
     if (this[$scene] != null) {
       this[$scene].dispose();
     }
+
     this[$scene] = new Scene(this[$engine]);
+    this[$scene].imageProcessingConfiguration.toneMappingEnabled = true;
+    this[$scene].imageProcessingConfiguration.toneMappingType =
+        ImageProcessingConfiguration.TONEMAPPING_ACES;
+    // tone mapping makes a shift on explosure, so it should be increased be
+    // some factor
+    this[$scene].imageProcessingConfiguration.exposure = 2.0;
```

Comment:
exposure = 1.8: (very simiar to 1.67, 1.4 is a little dark)
<img width="558" alt="1 8" src="https://user-images.githubusercontent.com/44676379/87114279-315a8c00-c23e-11ea-8ffa-596122682044.png">

exposure = 2.0
<img width="563" alt="2 0" src="https://user-images.githubusercontent.com/44676379/87114334-50f1b480-c23e-11ea-9861-983a75a334e2.png">

exposure = 2.2 (2.4 is obviously bright)
<img width="576" alt="2 2" src="https://user-images.githubusercontent.com/44676379/87114370-6535b180-c23e-11ea-8ea0-3193db19e92f.png">

Although 1.8 looks better in terms metric, but if feel like 2.0 looks closer to other renderers. Do you agree with it?

Reply:
Yeah, 2.0 looks pretty good to me. We'll be updating three.js and filament soon anyway, and they both contain tone mapping changes, so we'll probably have to revisit this anyway. This looks like a good baseline.